### PR TITLE
Making the website ugly, adding some sample emails

### PR DESCRIPTION
**Pull request by philyeung**

### CODE REVIEW

File: inbox/email_1.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Hey Dev Team!
```
Comment (KushTurner) [inline inbox/email_1.md:1]: Should we add dates to the emails to give some hints? eg "Sent 20/03/2024" (3 days after St Patrick's day)

File: inbox/email_1.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+I was just thinking, the go live of the product coincides with St Patrick's Day - this is a great opportunity to take
+advantage of some marketing. We should go live with a green background on the site!
+
+Thanks,
+
+Steve
+
+## Tips
+
+The tailwind.config.js contains the main default colouring styles for the website. Use a website like https://coolors.co/
```
Comment (KushTurner) [inline inbox/email_1.md]: I'll only leave it on this email but there's something useful you can do in markdown which is add a hyperlink to a file, for this one you can do something like `[tailwind.config.js](../tailwind.config.js)` and then you can navigate to the file by clicking on it. Maybe we can do it for all the emails?

Comment (KushTurner) [reply inbox/email_1.md]: It's not letting me show what I want to but here's the reference docs https://www.markdownguide.org/basic-syntax/#links

File: inbox/email_6.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+## Tips
+
+Sounds like we have some pretty opinionated requirements!
+
+Take a look at MainPage.jsx and at this code:
+
+    const moviesData = mockedMovies.Search
+
+Search is a list of movies. Arrays in JavaScript have a number of ready made functions available on them - for example,
```
Comment (KushTurner) [inline inbox/email_6.md]: Can we assume they will know what an array is? It is a simple data structure but not sure if they will know it if they aren't CS students.

File: inbox/email_J_3.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Hi Team,
+
+Can we reduce the size of the text that appears with the movies?
+
+Jeenal
+
+## Tips
+
+On MovieCard.jsx, can you find the code responsible for displaying the movie title and description and see how you could
+reduce the text size?
```
Comment (KushTurner) [inline inbox/email_J_3.md:10]: Should we add this useful reference guide to help them pick some options or let them experiment with intellisense?

https://tailwindcss.com/docs/font-size

**Review by KushTurner — CHANGES_REQUESTED** [review]

File: inbox/email_1.md (hunk 1 above)
Comment (philyeung) [reply inbox/email_1.md:1]: Yep sure - I had the dates originally but removed them, I thought I'd add them back in once we had all the emails added in from everyone then we can do some sorting based on chronological order and get dates added in

**Review by KushTurner — APPROVED** [review]

**Pull request merged**